doc/src/manual/methods.md

```diff
@@ -906,7 +906,7 @@ f(x::Int, y::Int) = 3
 ```
 
 This is often the right strategy; however, there are circumstances
-where following this advice blindly can be counterproductive. In
+where following this advice carelessly can be counterproductive. In
```

Comment:
`mindlessly`? It's more about acting without thinking about whether it is the correct solution than acting without care, though of course the meanings are close.